reanalysis/reanalysis_conf.json

```diff
@@ -0,0 +1,91 @@
+{
+    "moi_tests": {
+        "description": "thresholds for different filters during the MOI checks",
+        "gnomad_dominant": 0.001,
+        "gnomad_max_homs_dominant": 1,
+        "gnomad_max_homs_recessive": 10,
+        "gnomad_max_ac_dominant": 100,
+        "class_2_new_only": true
+    },
+    "variant_object": {
+        "description": "variables affecting how the VCF variants are parsed, and AnalysisVariant objects are populated",
+        "csq_string": "Allele|Consequence|SYMBOL|Gene|Feature|MANE_select|BIOTYPE|EXON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|ALLELE_NUM|VARIANT_CLASS|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|GENE_PHENO|SIFT|PolyPhen|LoF|LoF_filter|LoF_flags",
+        "var_info_keep": [
+            "AC",
+            "AN",
+            "cadd",
+            "clinvar_sig",
+            "clinvar_stars",
+            "eigen_phred",
+            "exac_af",
+            "exac_ac_hemi",
+            "exac_ac_het",
+            "exac_ac_hom",
+            "fathmm",
+            "gerp_rs",
+            "gnomad_ac",
+            "gnomad_af",
+            "gnomad_an",
+            "gnomad_cov",
+            "gnomad_hom",
+            "gnomad_ex_ac",
+            "gnomad_ex_af",
+            "gnomad_ex_an",
+            "gnomad_ex_cov",
+            "gnomad_ex_hom",
+            "metasvm",
+            "mutationtaster",
+            "phast_cons",
+            "revel",
+            "slivar_comphet",
+            "splice_ai_csq",
+            "splice_ai_delta"
+        ]
+    },
+    "hail": {
```

Comment:
We're using Hail to implement this stage, but would it make sense to rename this stage to describe its purpose instead (e.g. `filter`)?